src/app/components/Billboard/index.test.tsx

```diff
@@ -141,6 +141,10 @@ describe('Billboard', () => {
   });
 
   it('should render an h2 heading with the text for "More on this" translated', () => {
+    // can remove this setting of the translation when pidgin moreOnThis translation is available
+    const originalMoreOnThis = pidginService.default.translations.moreOnThis;
+    pidginService.default.translations.moreOnThis = 'More on this';
+
```

Comment:
could we just ensure that we have a default english translation set in index.tsx, shouldn't we have english as a worst case fallback anyway? So something like this in index.tsx :


```
  const {
    moreOnThis = 'More on this'
  } = translations ?? {};

```

Reply:
Think we've decided not to show the title if there are no translations: https://bbc-tpg.slack.com/archives/C03RK72084S/p1751621112644579?thread_ts=1751558609.633789&cid=C03RK72084S

Reply:
ah cool makes sense !

Reply:
sorry didn't refresh the page, so didn't realise Aaron had already replied!